rust/src/sip/detect.rs

```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2019 Open Information Security Foundation
+/* Copyright (C) 2024 Open Information Security Foundation
```

Comment:
nit: when updating copyright years, we add the current year beside the initial one:
```suggestion
/* Copyright (C) 2019-2024 Open Information Security Foundation
```